tests/test_schema.py

```diff
@@ -96,17 +96,17 @@ def test_repr():
     without_escape = ANSI_ESCAPE.sub("", out_repr)
     assert (
         without_escape.replace("\r", "")
-        == """╭─────────────┬─────────╮
-│ Column Name ┆ Type    │
-╞═════════════╪═════════╡
-│ int         ┆ Int64   │
-├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
-│ float       ┆ Float64 │
-├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
-│ string      ┆ Utf8    │
-├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
-│ bool        ┆ Boolean │
-╰─────────────┴─────────╯
+        == """╭────────┬─────────╮
+│ column ┆ type    │
```

Comment:
I'd prefer `column_name` over `column`, but if others have strong opinions on `column`, i'd be fine keeping it as is.